src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestUrlBuilder.java

```diff
@@ -330,16 +327,12 @@ public void urlWithLanguageBuilderTests() throws Exception {
               (String) data[10])
               .getUrlForPath((String) data[1]),
           (String) data[8]);
-      Assert.assertEquals(
-          c.newInstance((String) data[0], "staging", (Boolean) data[2], (Boolean) data[3],
-              (String) data[10])
-              .getUrlForPath((String) data[1]),
-          (String) data[9]);
     }
   }
 
   @Test(groups = {"TestUrlBuilder", "unitTests"})
   public void getWikiaUrlTests() throws Exception {
+    Configuration.setTestValue("forceFandomDomain", "false");
     Constructor<UrlBuilder> c = getUrlBuilderConstructor();
 
     Assert.assertEquals(c.newInstance("mediawiki119", "preview", false, false, DEFAULT_LANGUAGE)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "mediawiki119" 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)